app/controllers/votes_controller.rb

```
@@ -1,29 +1,33 @@
 class VotesController < ApplicationController
-    # Vote on a question
-    def vote_on_question
-      vote = Vote.new(vote_params)
-      vote.votable = Question.find_by(id: params[:question_id])
-      save_vote(vote)
-    end
+  skip_before_action :verify_authenticity_token, only: [ :vote_on_question, :vote_on_answer ]
 
-    # Vote on an answer
-    def vote_on_answer
-      vote = Vote.new(vote_params)
-      vote.votable = Answer.find_by(id: params[:answer_id])
-      save_vote(vote)
-    end
+  # Vote on a question
+  def vote_on_question
+    vote = Vote.new(vote_params)
+    vote.votable = Question.find_by(id: params[:question_id])
```

we can use find here. let it fail if not found